crates/sql-sink/Cargo.toml

```diff
@@ -15,7 +15,7 @@ async-std = { version = "1.12",  default-features = false, features = ["attribut
 serde = { version = "1.0", default-features = false, features = ["derive"]}
 serde_json = { version = "1", default-features = false }
 url = { version = "2.4", default-features = false, features = ["serde"] }
-itertools = { version = "0.10", default-features = false }
+itertools = "0.11"
```

Comment:
we are bringing in default features, is it intentional?

Reply:
yes apparently they put the `join` behind the std feature because it allocates

Reply:
or rather it seems like that feature was being turned on somehow, our code doesn't compile without the feature anymore

Reply:
it was being turned on because of `cranelift-wasm` which we pull in because `wasmtime`. Cargo just merges the features so we got that feature as well but now that we depend of `0.11` but `wastime` pulls in `0.10.5` we don't get that feature merged hence the code doesn't compile